File: Aula 3 - Modificadores de Acesso/projetoPython/src/novo/modificador.py
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.


class Cliente:
    
    def __init__(self):
        self.atrPublico = "1"
        self._atrProtected = "2" #Atributo Protected por Convenção
        self.__atrPrivate = "3"

File: Aula 3 - Modificadores de Acesso/projetoKotlin/src/main/kotlin/br/ufmt/projetokotlin/Modificador.kt
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package br.ufmt.projetokotlin

/*

  @author raphael
  Created on 07/04/2018
*/

open class Cliente {
    
    public var nome: String = "";
    
    public var idade: Int = 0
        protected set(value) {
            if (value >= 0) {
                field = value;
            }
        };
}